src/shared/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,36 @@
+import { NavLink } from 'react-router-dom';
+
+import IconAdd from '@components/svg/IconAdd';
+import IconHome from '@components/svg/IconHome';
+import IconUsers from '@components/svg/IconUsers';
+
+import './Navbar.scss';
+
+function Navbar() {
+  return (
+    <>
+      <div className="shadow-nav" />
+      <div className="navigation">
+        <ul>
+          <li>
+            <NavLink to="/" className={({ isActive }) => (isActive ? 'active' : 'not-active')}>
+              <IconHome />
+            </NavLink>
+          </li>
+          <li>
+            <NavLink to="/users" className={({ isActive }) => (isActive ? 'active' : 'not-active')}>
+              <IconUsers />
+            </NavLink>
+          </li>
+          <div className="add-button">
+            <NavLink to="/sentence-form">
+              <IconAdd />
+            </NavLink>
+          </div>
```

Comment:
Tu peux remplacer ta `<div />` en `<li />` car cet élément fait également partie de ta navigation. Tu peux rajouter une clé dans l'objet **PATHS** qui pourrait s'appeler **ADD_SENTENCE**. Comme il s'agit juste de l'ajout l'url pourrait être "/sentence-add" ou "/sentences/add".

```suggestion
          <li>
            <NavLink to={PATHS.ADD_SENTENCE}>
              <IconAdd />
            </NavLink>
          </li>
```